src/content/en/2019/compression.md

```diff
@@ -162,19 +162,23 @@ Most text based resources (such as HTML, CSS, and JavaScript) can benefit from g
 In the graph below, the top 25 content types are displayed with box sizes representing the relative number of requests. The color of each box represents how many of these resources were served compressed. Most of the media content is shaded red, which is expected since gzip and brotli would have little to no benefit for them.  Most of the text content is shaded green to indicate that they are being compressed. However, the light green shading for some content types indicate that they are not compressed as consistently as the others. 
 
 <figure>
-  <a href="/static/images/2019/15_Compression/ch15_fig2_compressed_content_types.jpg">
-    <img src="/static/images/2019/15_Compression/ch15_fig2_compressed_content_types.jpg" alt="" width="600">
+  <iframe aria-labelledby="fig3-caption" width="779.5" height="481.99083333333334" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQNIyMEGYE_1W0OdFYLIKsxg6M3o_ZsTTuaX73Zzv6Alw4x4D6oH0jdg9BSgw-jy4E-MmX_Qaf-B98W/pubchart?oid=1790056981&amp;format=interactive"></iframe>
+  <a href="/static/images/2019/15_Compression/fig3.png" class="fig-mobile">
+    <img src="/static/images/2019/15_Compression/fig3.png" aria-labelledby="fig3-caption" width="600">
   </a>
+  <div id="fig3-caption" class="visually-hidden">Figure 3. Top 25 compressed content types.</div>
```

Comment:
If someone can't see the results in the chart, what are the takeaways you'd want them to know?

Reply:
Just updated the chart in the google sheet.   It was incorrect.   Also updating the description above the chart, which is referencing the old color scheme. 


I've updated this to read: 
```
In the graph below, the top 25 content types are displayed with box sizes representing the relative number of requests. The color of each box represents how many of these resources were served compressed. Most of the media content is shaded orange, which is expected since gzip and brotli would have little to no benefit for them.  Most of the text content is shaded blue to indicate that they are being compressed. However, the light blue shading for some content types indicate that they are not compressed as consistently as the others. 
```